src/radio-tile/radio-tile.scss

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+@mixin iui-radio-tile {
+  cursor: pointer;
+  width: $iui-xl * 5;
+  height: 100%;
+  box-sizing: border-box;
+  padding: $iui-s;
+  position: relative;
+  z-index: 1;
+  border: 1px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
```

Comment:
Not sure whats the process now, but I thought we agreed to have IE support drop for version 1 only.
So these could be better wrapped under `themed` mixin, instead of using css vars directly.

Reply:
After our CSS 1.0 meeting this morning, I think we decided that this PR will be pre-1.0 so the CSS variables will need to go back to the `themed` `t()` variables.